main.py

```diff
@@ -1,12 +1,253 @@
+        postal_code=input("What is the customer's postal code? ")
+        phone_num=input("What is the customer's phone number? ")
+
+        response = customer_list.add_customer(name=name, postal_code=postal_code, phone_num=phone_num)
+
+        print_stars()
+        print("New customer", response["name"], "registered.")
+
+    elif choice=='8':
+        customer = select_customer(customer_list)
+
+        if customer:
+            if not customer_list.selected_customer:
+                print("Customer not found.")
+                return
+
+        print_stars()
+        print(f"Selected customer: {customer}")
+    
+    elif choice=="9":
+        customer = select_customer(customer_list)
+
+        if customer:
+            if not customer_list.selected_customer:
+                print("Customer not found.")
+                return
+                
+            if customer_list.selected_customer:
+                print(f"Great! Let's update the customer: {customer_list.selected_customer}")
+                
+                name=input("What is the new name of the selected customer? ")
+                postal_code=input("What is the new postal code of the selected customer? ")
+                phone_num=input("What is the new phone number of the selected customer? ")
+
+        response = customer_list.update_customer(name=name, postal_code=postal_code, phone_num=phone_num)
+
+        print_stars()
+        print("Updated customer number:", response["id"]) 
+
+    elif choice=='10':
+        customer = select_customer(customer_list)
+
+        if customer:
+        # Note from chats w/Becca & Chris: prompt user to delete each associated rental first (CLI) or delete rentals (instances of CustomerVideoRental) associated with customer (API)
+            if not customer_list.selected_customer:
+                print("Customer not found.")
+                return
+
+            if customer["videos_checked_out_count"] > 0:
+                print("Customer has outstanding videos and cannot be deleted.")
+                return
+
+            customer_list.delete_customer()
+
+            print_stars()
+            print("The customer has been deleted from inventory.")
+
+# Rental Related Choices ---------------------------------------------------------------------------
+
+    elif choice=='11':
+        print("Ready to check-out?")
+
+        customer_id=int(input("What is the id of the customer? "))
+        video_id=int(input("What is the id of the video? "))
+
+        response = rental_list.check_out(customer_id=customer_id, video_id=video_id)
+
+        print_stars()
+        print("Video checked out.  Due on: ", response["due_date"])
+
+    elif choice=='12':
+        print("Checking in a video?")
+
+        customer_id=int(input("What is the id of the customer? "))
+        video_id=int(input("What is the id of the video? "))
+
+        response = rental_list.check_in(customer_id=customer_id, video_id=video_id)
```

Comment:
We need to check what the response is. If it is a 400 or a 404, that means we did not successfully check in the video. Consider what behavior you want for a video that is not checked in

Reply:
For instance, something like:
```
if response.status_code != 200:
   print("Something went wrong")
else:
   print("Video checked in")
```